app/javascript/Greetings.js

```diff
@@ -0,0 +1,16 @@
+import React, { useEffect } from "react";
+import { useDispatch, useSelector } from "react-redux";
+import { showGreetings } from "./Redux/Greeting/greeting";
+
+const Greetings = () => {
+  const { greetings, loading } = useSelector((state) => state.message);
+  const dispatch = useDispatch();
+  useEffect(() => {
+    dispatch(showGreetings());
+  }, []);
+  return (
+    <div>{loading ? <h1>Loading...</h1> : <h1>{greetings.greeting}</h1>}</div>
```

Comment:
- [ ] Kindly make sure that  your app displays a random greeting, currently, only the loading message is shown when I load or refresh the app

![Screenshot_602](https://user-images.githubusercontent.com/69638013/211413385-a444aeab-93fa-43b1-9ff0-50061abc49d8.png)

Reply:
@prolajumokeoni I made feature to display Loading while api is fetching. You should wait at least few seconds and it will display random quotes. I am going to send you video of working project. Kindly ask you to check again app. Thank you :)

Reply:
Hi @edi-sipka, I waited but no random greeting was displayed only the loading state was shown